### PR TITLE
Refs #30129 -- Added test for create() with F() expression in Subquery.

### PR DESCRIPTION
**Pull request by sarahboyce**
Hello!
I think this [ticket](https://code.djangoproject.com/ticket/30129) has been resolved (I'm not sure when) and can be closed.
I don't think a test is required, this is just to reference on the ticket to check I haven't missed anything (I will close the PR).

I'm also aware I can just close tickets but not sure I'm supposed to as it feels like that's a tightly managed field, let me know if I should just close in future :+1:

### CODE REVIEW

**Comment by felixxm** [comment]
@sarahboyce Thanks for this patch :+1: 

> I don't think a test is required, this is just to reference on the ticket to check I haven't missed anything (I will close the PR).

New tests are always required to confirm new features.

> I'm also aware I can just close tickets but not sure I'm supposed to as it feels like that's a tightly managed field, let me know if I should just close in future +1

We will close the ticket when the tests are merged.

File: tests/expressions/models.py [hunk 1]
```diff
@@ -102,3 +102,15 @@ class UUIDPK(models.Model):
...
+class Item(models.Model):
+    pass
+
+
+class ItemVersion(models.Model):
+    item = models.ForeignKey(Item, related_name="versions", on_delete=models.CASCADE)
+    version_number = models.IntegerField(default=0)
+
+    class Meta:
+        unique_together = ("item", "version_number")
```
Comment (felixxm) [inline tests/expressions/models.py:116]: Can we reuse existing models?

As far as I'm aware `unique_together` is not required :thinking:

Comment (sarahboyce) [reply tests/expressions/models.py]: Updated to have the one test with existing models - I think it covers the ticket requirement :+1:

File: tests/expressions/tests.py [hunk 2]
```diff
@@ -381,6 +381,24 @@ def test_object_update_unsaved_objects(self):
         with self.assertRaisesMessage(ValueError, msg):
             acme.save()
 
+    def test_ticket_30129_insert_f_expression_in_subquery(self):
```
Comment (felixxm) [inline tests/expressions/tests.py:384]: Ticket numbers are unnecessary in the test method names:
```suggestion
    def test_insert_f_expression_in_subquery(self):
```

**Review by felixxm — COMMENTED** [review]
@sarahboyce Thanks :star2:

**Pull request merged**